### PR TITLE
Modified the Join and Leave Routes

### PR DESCRIPTION
**Pull request by Rishi-121**
Screenshot (107)

### CODE REVIEW

**Review by mohit355 — CHANGES_REQUESTED** [review]
Upload all the local files as well that you used for testing.

**Comment by mohit355** [comment]
Upload some snapshot of the UI

File: server/controllers/classrooms.js [hunk 1]
```diff
@@ -43,29 +39,48 @@ const Create = async (req, res) => {
...
+
+  // Get the classroom details
+  const classroom = await Classroom.findById({ code: classroomCode });
+  if (!classroom) {
+    return res.status(404).send(`${classroomCode} is an invalid code!`);
+  } else {
+    // Update the classroom by adding a new user
+    const updatedClass = await Classroom.findByIdAndUpdate(
+      classroomCode,
+      { $push: { users: user } },
```
Comment (aavishkarmishra) [inline server/controllers/classrooms.js:53]: Please define `user` before using it and check that apis are working or not.

File: server/routes/classrooms.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 
 const Classrooms = require("../controllers/classrooms");
+const userAuth = require("../middlewares/userAuth");
```
Comment (vikhyatsingh123) [inline server/routes/classrooms.js:4]: have you uploaded userAuth.js file?

**Comment by Rishi-121** [comment]
No, I am not working on this project.

**Pull request closed without merging**